package task01;

public class MainException extends Exception{

	public MainException() {  // 예외 발생시 동작하는 메소드
		System.out.println("잘못된 입력입니다");  // 메시지 출력
	}
	
}
